zerver/models.py

```diff
@@ -771,6 +779,20 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
         wildcard_mention_policy=int,
     )
 
+    REALM_PERMISSION_GROUP_SETTINGS: Dict[str, GroupPermissionSetting] = dict(
+        create_multiuse_invite_group=GroupPermissionSetting(
+            require_system_group=True,
+            allow_internet_group=False,
+            allow_owners_group=False,
+            allow_nobody_group=True,
+            default_group_name=MODERATORS_GROUP_NAME,
+        ),
+    )
+
+    REALM_PERMISSION_GROUP_SETTING_ID_TO_GROUP_SETTING = dict(
+        create_multiuse_invite_group_id="create_multiuse_invite_group"
```

Comment:
I get how this is used, but we can directly add "_id" to the setting name. I feel it is unnecessary, but we can wait for more feedback.

Reply:
I'm not opposed to this existing; I guess most places that currently reference `REALM_PERMISSION_GROUP_SETTING_ID_TO_GROUP_SETTING` feel like they should access `REALM_PERMISSION_GROUP_SETTINGS` instead, but maybe that doesn't work because of the `_id` suffix. 

I think the right larger model would be to have `GroupPermissionSetting` have an `id_field_name` field.

Reply:
Most of the places we use the `REALM_PERMISSION_GROUP_SETTING_ID_TO_GROUP_SETTING` object in a loop. So in those places we can just loop over `REALM_PERMISSION_GROUP_SETTINGS` and add "_id" to the string like we do for stream settings currently.

And a couple of other places where we use this differently is in `has_permission` code in `models.py` and in `views/realm.py` to check whether a setting is a group-based settings to add its new value in `req_group_setting_vars` object.

We can actually directly use `create_multiuse_invite_group` instead of `create_multiuse_invite_group_id` in `has_permission` as we eventually pass user group object in `is_user_in_group`. While this dict is actually helpful to use in the `views/realm.py` code.

Reply:
But yeah, I agree that having `id_field_name` would be a better plan.

Reply:
Hmm, keeping this dict is helpful in `views/realm.py`. If we will change the parameters to not have "_id" than we can remove this dict. But in that case I think we should also make changes in name of `page_params` and `events` because our current logic in frontend for changing the settings is based on the assumption that name of the setting received in `page_params` is same as name of the parameter used for changing the setting. Otherwise we need to figure out some approach to distinguish between role based and group based setting in frontend and  may also need to maintain a dict similar to `REALM_PERMISSION_GROUP_SETTING_ID_TO_GROUP_SETTING` in frontend.